frontend/www/js/omegaup/components/problem/Form.test.ts

```diff
@@ -15,7 +15,7 @@ const props: types.ProblemFormPayload = {
   statusError: '',
   allowUserAddTags: true,
   showDiff: 'none',
-  timeLimit: 1000,
+  timeLimit: '',
```

Comment:
```suggestion
  timeLimit: 1000,
```

huh, el tiempo límite debería ser un número, no una cadena. no se por qué TypeScript no se está quejando :S